app/controllers/api/v0/auctions_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V0::AuctionsController < ApplicationController
```

Comment:
thoughts on creating an `ApiController` that all api-related controllers inherit from? That we could slowly move shared api-related API logic into that.